### PR TITLE
Added 'May Ball' to title.

### PR DESCRIPTION
**Pull request by mattleeuk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Christ's 2020</title>
+    <title>Christ's May Ball 2020</title>
```
Comment (drsphelps) [inline index.html:6]: WHAT?

Comment (drsphelps) [reply index.html:6]: Step off it boss man

**Review by drsphelps — COMMENTED** [review]
Fuck off

File: index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Christ's 2020</title>
```
Comment (anik545) [inline index.html:6]: Excuse me??!?

**Pull request closed without merging**